internal/jimm/jimm.go

```diff
@@ -268,6 +199,70 @@ type SSHManager interface {
 	ResolveAddressesFromModelUUID(ctx context.Context, modelUUID string) ([]string, error)
 }
 
+// JujuManager is the interface to manage all Juju related operations.
+type JujuManager interface {
```

Comment:
This interface is the large one from `jujuAPI` but only includes methods for the Juju manager.